types/vote_set.go

```diff
@@ -163,9 +163,7 @@ func (voteSet *VoteSet) addVote(vote *Vote) (added bool, err error) {
 	blockKey := vote.BlockID.Key()
 
 	// Ensure that validator index was set
-	if valIndex < 0 {
-		return false, errors.Wrap(ErrVoteInvalidValidatorIndex, "Index < 0")
```

Comment:
should we remove ErrVoteInvalidValidatorIndex too? or it's used somewhere else?

Reply:
its still used elsewhere